packages/fast-components-react-base/src/breadcrumb/README.md

```diff
@@ -0,0 +1,8 @@
+## Breadcrumb
+A generic breadcrumb component following the w3.org [breadcrumb](https://www.w3.org/TR/2017/NOTE-wai-aria-practices-1.1-20171214/examples/breadcrumb/index.html) pattern. The *breadcrumb* will accept any child elements but is built to work with `a` elements.
+
+### Usage
+To denote the current page the *breadcrumb* automatically sets the last child `aria-current` prop to "page". Addtionally *breadcrumb* accepts an optional `separator` prop that will render a ReactNode between navigation elements. To support `rtl` in jss you need to target the `breadcrumb_ol` class for *breadcrumb* and set it to `display: "flex", flexWrap: "wrap"`
+
+### Accessibility
+The *breadcrumb* component consists of a `nav` element with `aria-label` set to a `label` prop. One should set the `label` prop to "Breadcrumb" to identify the structure as a breadcrumb trail and make it a navigation landmark so that it is easy to locate. The last child element `aria-current` prop is marked "page" to indicate that it reprents the current page.
```

Comment:
>  One should set the `label` prop to "Breadcrumb"

I don't think we need to be this specific. We should call out why an aria-label is required, not what the content should be